student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super()
+    @classroom = classroom
+  end
```

Comment:
- [ ] Good job Inheriting the `Student` class from `Person`. But you should ensure that the `Constructor` extends parent's constructor by adding `@specialization `and a `parameter` for it i.e.:
```ruby
  ...
  def initialize(age, classroom, name = "Unknown", parent_permission = true)
   super(age, name, parent_permission)
    ...
  end
  ...
```